### ISSUE TITLE
Tutorial_digging

### ISSUE THREAD

**Issue by napiorek** [PlaceholderGames/2016-VnD_Game#53]
The aim is to create a single game level where we can introduce digging(using shovel) feature to user as part of the game/tutorial.

**Comment by napiorek** [comment]
I will create the branch as soon as my previous issue (#41 )will be merge with develop. @BojanStankovic

**Comment by napiorek** [comment]
While creating the tutorial map for digging I realize that ant is bouncing off the cube that it should dig through. So this is preventing the ant form stepping over the cube so the Ant currently can't dig vertically ( down ) if it can't climb on that cube first ?

**Comment by napiorek** [comment]
Anyone have any suggestion how or if we can implement digging if the Ants are bouncing of the digging box ?

**Comment by napiorek** [comment]
I know that you guys were working on digging (@rbakerusw) and Ant IA (@ScaryEme).
I know that the Ant should collide with the DiggingBP box, but can we create separate material so the Ant would not to collide with ?

**Comment by napiorek** [comment]
I cloned the Cube_BP from DiggingBP folder and just break the links to disable collisions.
![image](https://cloud.githubusercontent.com/assets/22525895/23624718/a7ac9186-029e-11e7-9952-4a5330b0665b.png)

It works, but do we need to have a separate trigger box as well ?

**Comment by elliotnaylor** [comment]
Could you explain this to me when I see you next? I think it'll be easier to sort in person

**Comment by napiorek** [comment]
The trouble is that we need the Tutorial lvl for next game demo ...

**Comment by napiorek** [comment]
After intensive testing I conclude that now Ants falling through the box :(

**Comment by elliotnaylor** [comment]
You need to set the block collision to custom and select to block type ant.  This is fixed in one of my unmerged builds

**Comment by BojanStankovic** [comment]
Is this one solved than @napiorek and @ScaryEme. Looks to me like it is but don't want to close it because I'm not sure. If it is sorted and merged from somewhere else just reference it to that issue and close this one

**Comment by napiorek** [comment]
Maps done
 I'v introduce digging to the player on two occasions, first where there is no real lost if player will missed the ant or misused the digging feature(ants will just go past the digging cube and bounce off the wall and comeback) and the aim here is to give the player some experience of digging feature.
On the second occasions the player is require to be more precise, if not, then he might loose some ants by not acting quickly or precisely enough.

**Comment by BojanStankovic** [comment]
The map was mistakenly done inside of the ThirdPersonExampleMap and the DiggingTutorial_map was just a copy of the example map. I duplicated the example map in the branch #53 with the correct name and place it in the develop branch instead of the non-modified digging tutorial map.

Additionally I have discovered a bug with the ant spawner that doesn't direct the ants as it was supposed to do. I'll make a separate bug report for this.

This issue is solved and the branch was merged. Closing
